eng/G01_ABC.md

```diff
@@ -108,8 +108,6 @@ depending on what is most comfortable for them.
 
 A semivowel is a sound that is phonetically similar to a vowel sound but functions like a consonant as the syllable boundary.
 Pandunia has two semivowels **y** and **v**.
-They appear only in the beginning of a syllable
-and they are always followed by a full vowel.
```

Comment:
What made you think this is not true anymore?

Reply:
FYI the word that makes this not true anymore is *evri*.  As far as I know it's the only such word, so if you wanted to reinstate this rule, you could change it to *euri* or *hol un* or *ich* etc.

Reply:
You're right. It's the only one. One guy reminded me of the schwa insertion rule, which can make _evri_ to be pronounced /'evəri/. In spelling _evri_ would still be an exception, maybe, but I think that we can live with one. So I will reinstate this deleted rule.